src/OpenTelemetry/BatchExportProcessorOptions.cs

```diff
@@ -0,0 +1,34 @@
+// <copyright file="BatchExportProcessorOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using System.Threading;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry
+{
+    public class BatchExportProcessorOptions
+    {
+        public int MaxQueueSize { get; set; } = BatchExportProcessorDefaultOptions.MaxQueueSize;
```

Comment:
Do we need units on the two size props? Or maybe just some renaming like `MaxQueueLength`/`MaxBatchCount`?

Reply:
I'll rename it to MaxQueueLength. For, MaxBatchSize vs MaxBatchCount, I feel "size" goes well with batch as opposed to "count". I have no strong preference though.

Reply:
@CodeBlanch @utpilla Just a question for my own edification... what has been our precedent for naming things same/similar to names suggested in the spec? Have we been somewhat flexible in this regard?

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk.md#batching-processor

Reply:
I have renamed the BatchExportProcessorOptions properties to match the SDK spec since the file is now moved to OpenTelemetry and is not under OpenTelemetry.Exporter.Zipkin.

Reply:
My default choice is - pick what spec says, so it'll be consistent across all other implementations. If the spec name doesn't make sense, send a proposal to spec repo - its usually fast to get approvals if the proposed name is better at clarifying things.